ui/app/AppLayouts/Onboarding2/OnboardingStackView.qml

```diff
@@ -6,6 +6,9 @@ import StatusQ.Core.Theme 0.1
 StackView {
     id: root
 
+    readonly property bool backAvailable: currentItem ? (currentItem.backAvailableHint ?? true)
```

Comment:
I'm not sure if it's better place because it's very specific property related not to the component itself but to the way how it's used in our context of `OnboardingLayout`. It's kind of indirect coupling of this component, back button and pages.

Reply:
It's convenient to have it here imho (and optional); we'll definitely need it when we start constructing the "flows inside popups" stuff, like the new BackupSeedPhrase standalone dialog

Reply:
Yeah, I know it's convenient to keep it here. Just having a notion it's not fully architecturally clean, but it's probably a bit of an obsession of mine. Feel free to ignore that comment ;)